app/src/components/report/Table.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState } from 'react';
+import EditIcon from '@mui/icons-material/Edit';
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import InfoIcon from '@mui/icons-material/Info';
+import WarningIcon from '@mui/icons-material/Warning';
+import CheckIcon from '@mui/icons-material/Check';
+import AutorenewIcon from '@mui/icons-material/Autorenew';
+import PauseCircleIcon from '@mui/icons-material/PauseCircle';
+import CancelIcon from '@mui/icons-material/Cancel';
+import QuestionMarkIcon from '@mui/icons-material/QuestionMark';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import { Collapse, IconButton, Chip, ChipProps } from '@mui/material';
+import { IReport } from '@/models';
+import ButtonLink from '../ButtonLink';
+import Box from '@mui/material/Box';
+import { StatusOfReport, ReportCategories, IReportFields } from '@/models';
+import {
+    DataGrid,
+    GridRowsProp,
+    GridColDef,
+    GridActionsCellItem,
+    GridRowParams,
+    GridRenderCellParams,
+} from '@mui/x-data-grid';
+
+const expandedRowStyle = {
```

Comment:
By convention, constants should be named in `UPPER_SNAKE_CASE` like `EXPANDED_ROW_STYLE`